cosmic_ray/testing/test_runner.py

```diff
@@ -66,4 +68,4 @@ def __call__(self):
                                   test_result[1])
         except Exception as e:
```

Comment:
I'm currently running landscape on each commit to master, I think. At least, I've got a badge telling my I'm 83% healthy! I'll admit that I don't pay attention to those numbers very often, though.